tensorboard/webapp/notification_center/BUILD

```diff
@@ -23,3 +23,13 @@ tf_ts_library(
     ],
     deps = [],
 )
+
+tf_ng_web_test_suite(
+    name = "notification_center_test",
+    testonly = True,
+    srcs = [],
+    visibility = ["//tensorboard:internal"],
```

Comment:
don't need these two, I believe.

Reply:
![image](https://user-images.githubusercontent.com/2547313/111343376-9a4d9b80-8638-11eb-8900-0765ea326bfd.png)

Yup, you need to remove `srcs` here.